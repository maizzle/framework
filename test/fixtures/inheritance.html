---
template: second
---

<extends src="test/stubs/template.html">
  <block name="button">Child in second.html</block>
</extends>
