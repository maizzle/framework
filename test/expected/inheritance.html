Parent
    Child in second.html
